tendrl/monitoring_integration/grafana/notification_channel.py

```diff
@@ -11,7 +12,8 @@
 
 
 def create_notification_channel():
-    url = "http://" + str(NS.node_context.fqdn) + \
+    ip = socket.gethostbyname(socket.gethostname())
+    url = "http://" + str(ip) + \
           ":" + str(PORT) + "/grafana_callback"
```

Comment:
As per current implementation server fqdn is always None

Reply:
Ack